pkg/framework/provider/crypto/crypto_test.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package crypto
+
+import (
+	"testing"
+
+	chacha "golang.org/x/crypto/chacha20poly1305"
+
+	"github.com/google/tink/go/aead"
+	"github.com/google/tink/go/keyset"
+	"github.com/google/tink/go/signature"
+	aeadsubtle "github.com/google/tink/go/subtle/aead"
+	"github.com/stretchr/testify/require"
+
+	cryptoapi "github.com/hyperledger/aries-framework-go/pkg/framework/spi/crypto"
+)
+
+func TestCrypto(t *testing.T) {
+	// Assert that Crypto implements the Crypto interface.
+	var _ cryptoapi.Crypto = (*Crypto)(nil)
+}
+
+func TestCrypto_EncryptDecrypt(t *testing.T) {
+	// tests XChacha20Poly1305
```

Comment:
Using subtests for each case would be clearer.

Reply:
done